pattern-distributed/sp/wso2sp-worker-service.yaml

```diff
@@ -27,9 +27,13 @@ spec:
       port: 9090
       protocol: TCP
     -
-      name: 'https'
+      name: 'https-1'
```

Comment:
@pcnfernando shall we rename the port names as `https-port-{n}`?

We had followed a similar approach in other K8s resources as well (e.g. [1])

[1]: [Sample](https://github.com/wso2/kubernetes-is/blob/master/is-with-analytics/is-analytics/identity-server-analytics-2-service.yaml#L32)

Reply:
fixed with 233a5f92a7cf1e91cf11c2c2d1fae91680163f3c